cache/hhvm-scalable-cache.h

```diff
@@ -222,6 +222,14 @@ struct ConcurrentScalableCache {
    * if not outperform too much, don't do FH */
   double FH_PERFORMANCE_THRESHOLD = 0.2;
 
+  // Dynamic periodic refresh
+  int MIN_REFRESH_PERIOD = 10;
```

Comment:
We might want to tie the `MIN_REFRESH_PERIOD` to the user provided `rebuild_freq`?

Reply:
Not opposed to it, but I think the 10-100 range was supplied from the paper as being the measured range of optimal refresh period sizes based on the workload